xilem/src/lib.rs

```diff
@@ -273,12 +271,6 @@ impl ViewCtx {
         }
     }
 
-    pub fn mark_changed(&mut self) {
-        if cfg!(debug_assertions) {
-            self.view_tree_changed = true;
-        }
-    }
-
```

Comment:
I mean I agree with that change, but it doesn't seem related to the title of this PR?

Reply:
The idea is Xilem now uses `needs_rewrite_passes` to check the same thing.